### DISCUSSION TITLE
Shift Crouching can change Armour Stand legs and arms positions.

### DISCUSSION THREAD

**Discussion by Tydium** [PurpurMC/Purpur#483] [__Dismissed__] [answered]
I had this idea while playing bedrock edition. Basically when you shift and right click you can change the armour stands into a certain position, this with redstone also will trigger that. It would trigger into 5 (i think) preset positions. I know Java edition is able to edit the arms and legs via commands. So I think this may be possible. Just up to you. If you want the preset Armour stands in bedrock then ill send screenshots.

**Comment by BillyGalbreath** [comment, marked as answer]
Duplicate of https://github.com/pl3xgaming/Purpur/discussions/479
